tests/FluentBuilderGeneratorTests/DTO/UserWithParameterizedConstructor.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace FluentBuilderGeneratorTests.DTO;
+
+public class UserWithParameterizedConstructor
```

Comment:
Can you rename this to `ThingWithParameterizedConstructor` ?

I want to use all `Thing***` classes for testing constructors.